stackslib/src/chainstate/stacks/db/blocks.rs

```diff
@@ -9444,49 +9440,50 @@ pub mod test {
         assert_block_stored_not_staging(&mut chainstate, &consensus_hashes[0], blocks[0]);
 
         // process and store blocks 1 and N, as well as microblocks in-between
-        for (i, block) in blocks.iter().skip(1).enumerate() {
+        let len = blocks.len();
+        for i in 1..len {
```

Comment:
Oh, I see what the problem was:

```suggestion
        for (i, block) in blocks.iter().enumerate().skip(1) {
```